pages/changelog.js

```diff
@@ -92,18 +92,18 @@ export default function Changelog() {
         description="What are the latest features and changes to LinkFree"
       />
       <Page>
-        <h1 className="text-4xl mb-4 font-bold">Changelog</h1>
+        <h1 className="mb-4 text-4xl font-bold">Changelog</h1>
```

Comment:
I think this change is not required?

```suggestion
        <h1 className="text-4xl mb-4 font-bold">Changelog</h1>
```